orchestration/dagster_orchestration/hca_orchestration/resources/config/scratch.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+
+from dagster import InitResourceContext, resource, String, Int
+
+
+@dataclass
+class ScratchConfig:
+    scratch_bucket_name: str
+    scratch_prefix_name: str
+    scratch_bq_project: str
+    scratch_dataset_prefix: str
+    scratch_table_expiration_ms: int
+
+
+@resource({
+    "scratch_bucket_name": String,
+    "scratch_prefix_name": String,
+    "scratch_bq_project": String,
```

Comment:
These fields will be needed by the data file load code, so rather than repeating solid config, I'm centralizing in this resource config.